packages/core/src/editor/useInternalEditor.ts

```diff
@@ -35,8 +37,13 @@ export type useInternalEditorReturnType<C = null> = useCollectorReturnType<
 export function useInternalEditor<C>(
   collector?: EditorCollector<C>
 ): useInternalEditorReturnType<C> {
-  const handlers = useEventHandler();
   const store = useContext(EditorContext);
+  invariant(
+    Object.keys(store).length > 0,
```

Comment:
Why not `invariant(store, ERROR....)`?

Reply:
The [default](https://github.com/prevwong/craft.js/blob/develop/packages/core/src/editor/EditorContext.tsx#L6) is an empty object. I didn't want to change that. That's why take the number of keys into account.

Reply:
Can we make it the default into `undefined` or `null`? Don't think that would change things too much.

Reply:
You are right it did not change anything 👍